README.md

```diff
@@ -2,33 +2,109 @@
 
 [![Build Status](https://travis-ci.com/PaNOSC-ViNYL/libpyvinyl.svg?branch=master)](https://travis-ci.com/PaNOSC-ViNYL/libpyvinyl)
 [![Documentation Status](https://readthedocs.org/projects/libpyvinyl/badge/?version=latest)](https://libpyvinyl.readthedocs.io/en/latest/?badge=latest)
```

Comment:
Can you make sure,rtd works as intended?

Reply:
Working on that.

Reply:
@CFGrote Could you grant me management access to the RTD?

Reply:
I checked https://libpyvinyl.readthedocs.io/en/develop/, the [DEMO](https://libpyvinyl.readthedocs.io/en/develop/include/notebooks/example-01.html#Implement-a-Calculator-that-derives-from-BaseCalculator.) seems not working. Should we aim for fixing it in PR #49?